Basic-Scripts/balanced_paranthesis.py

```diff
@@ -0,0 +1,26 @@
+open_bracket_list = ["[", "{", "("]

```

Comment:
Follow [Global Variable Guidelines](https://www.python.org/dev/peps/pep-0008/#global-variable-names) for naming convention.